vocabularies/Common.xml

```diff
@@ -113,6 +113,17 @@
         </Member>
       </EnumType>
 
+      <Term Name="Timezone" Type="Edm.String" AppliesTo="Property Parameter">
+        <Annotation Term="Common.Experimental" />
+        <Annotation Term="Core.Description" String="The point in time represented by the annotated property or parameter shall be interpreted in the given time zone" />
+        <Annotation Term="Core.LongDescription">
+          <String>The value list for time zones is not prescribed by this vocabulary.
+            If an [IANA](https://www.iana.org/time-zones) time zone is specified, browsers can perform the necessary conversion.
```

Comment:
Would it not be possible to request that the time zone must be an IANA time zone? In the end, we want to interpret it in the browser. So what help would other time zones be?

Reply:
For example, browsers also understand CET. Or in the future, they may understand other standards.

Reply:
So, the gracious renderer has to understand it to be able to convert. Otherwise it's only for visualization purposes? And it's not the "browser" that performs the conversion - it has to be client-side code (that might run in the browser).

Reply:
> And it's not the "browser" that performs the conversion - it has to be client-side code (that might run in the browser).

The necessary Javascript code is one line, but the real work is done by the browser (or perhaps the operating system).

Reply:
It seems that CET is a deprecated IANA time zone. In the process of consuming this annotation we will need to find out whether all ABAP time zones are actually deprecated IANA time zones or just some.